build/build_deploy.sh

```diff
@@ -8,13 +8,17 @@ CURRENT_DIR=$(dirname "$0")
 
 BASE_IMG="managed-cluster-validating-webhooks"
 QUAY_IMAGE="quay.io/app-sre/${BASE_IMG}"
-IMG="${BASE_IMG}:latest"
+IMG="${BASE_IMG}:${VERSION_MAJOR}.${VERSION_MINOR}-${GIT_HASH}"
+VERSION_MAJOR=0
+VERSION_MINOR=1
 
 GIT_HASH=$(git rev-parse --short=7 HEAD)
 
 # build the image
 BUILD_CMD="docker build" IMG="$IMG" make docker-build
 
+GEN_SYNCSET=$(generate_syncset.py -t "${SELECTOR_SYNC_SET_TEMPLATE_DIR}" -y "${YAML_DIRECTORY}" -d "${SELECTOR_SYNC_SET_DESTINATION}" -r "${REPO_NAME}")
```

Comment:
Is this line meant to be here? I'm not seeing where these variables are set.

Reply:
Variable definition added, thanks for the initial review/comment :+1:

Reply:
Would it make sense to check the return of the generate_syncset.py call, and explicitly error out the entire process in case it fails

Reply:
I tried running the build_deploy.sh script manually and got a `build/build_deploy.sh: line 23: generate_syncset.py: command not found`
Should call the generate_syncset.py with the full path? including `build/`